Doppler.ContactPolicies.Api/Controllers/ContactPolicies.cs

```diff
@@ -30,5 +31,21 @@ public async Task<IActionResult> GetContactPoliciesSettings(string accountName)
 
             return new OkObjectResult(contactPoliciesSettings);
         }
+
+        [Authorize(Policies.OWN_RESOURCE_OR_SUPERUSER)]
+        [HttpPut("/accounts/{accountName}/settings")]
+        public async Task<IActionResult> UpdateContactPoliciesSettings(string accountName,
+            [FromBody] ContactPoliciesSettingsDto contactPoliciesSettings)
+        {
+            var newContactPoliciesSetting =
+                await _contactPoliciesService.InsertContactPoliciesSettingsAsync(accountName, contactPoliciesSettings);
```

Comment:
Why do you call it `Insert...` in place of `Update...`? 

I mean, it is not an insertion, right? or I am missing something?

Reply:
it can be call as Updated, i think i get confuse at the time of calling the query as always is an insertion

Reply:
Right, if I am not wrong, this method does three things:

* Updates the UserShipingLimit
* Deletes the removed excluded lists
* Adds the added excluded lists

So, probably `Update` is a better word. Also, `update`'s semantics is idempotent (`insert` is not), so If I am not wrong it describes the desired behavior better.

Reply:
I think that we agree that it is an "update" operation, so I am marking it as resolved.

Let me know if you want to discuss the independent behavior that I am proposing.